data/json/itemgroups/Weapons_Mods_Ammo/guns.json

```diff
@@ -548,6 +548,7 @@
       { "item": "remington700_270", "prob": 25 },
       { "item": "ruger_m77", "prob": 10 },
       { "item": "ruger_arr", "prob": 10 },
+      { "item": "steyr_scout", "prob": 10 },
```

Comment:
Continuing from Discord discussions...

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/7764202/6628a787-f162-4fb9-82d0-dd288cb7de57)
On gunbroker I've found 350 hits for completed listings of the steyr scout in .308. 

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/7764202/d582e25a-ad22-43d5-a841-011a500c391b)
I've found 361 completed listings of the Keltec RFB.

Since I've found ~6000 listings completed of the savage 110 (which the 111F is related to) I recommend moving the steyr scout entries over from common guns item groups and into rare guns item groups, and multiplying the RFB's spawn weights by (350/361) to get your Steyr Scout spawn weights.
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/7764202/0fa921a4-0c89-4483-9740-224c873e7de0)